bluepysnap/edges/edge_population_stats.py

```diff
@@ -0,0 +1,79 @@
+"""EdgePopulation stats helper."""
+
+import numpy as np
+
+from bluepysnap.exceptions import BluepySnapError
+
+
+class StatsHelper:
+    """EdgePopulation stats helper."""
+
+    def __init__(self, edge_population):
+        """Initialize StatsHelper with an EdgePopulation instance."""
+        self._edge_population = edge_population
+
+    def divergence(self, source, target, by, sample=None):
+        """`source` -> `target` divergence.
+
+        Args:
+            source (int/CircuitNodeId/CircuitNodeIds/sequence/str/mapping/None): source nodes
+            target (int/CircuitNodeId/CircuitNodeIds/sequence/str/mapping/None): target nodes
+            by (str): 'synapses' or 'connections'
```

Comment:
I think a bit more clarity of what `synapses` and `connections` means; perhaps an small example?

Reply:
Sure thing. Added an explanation in the docstrings